src/schemaGenerator.ts

```diff
@@ -69,6 +69,7 @@ function _generateSchema(
   allowUndefinedInResolve: boolean,
   resolverValidationOptions: IResolverValidationOptions,
   parseOptions: GraphQLParseOptions,
+  inheritResolversFromInterfaces: boolean
```

Comment:
This boolean argument is fine because `_generateSchema` is private to this module (and not exported like `addResolveFunctionsToSchema`).